ggshield/verticals/sca/file_selection.py

```diff
@@ -32,6 +31,10 @@
     ".hypothesis",
 )
 
+SCA_EXCLUSION_REGEXES = {
+    re.compile(r".*" + pattern + ".*") for pattern in SCA_IGNORE_LIST
```

Comment:
```suggestion
    re.compile(r".*/" + re.escape(pattern) + "/.*") for pattern in SCA_IGNORE_LIST
```
Some improvements over what I initially suggested
- add `/` to separate paths
- use `re.escape` , otherwise `.` in patterns get interpreted as regex patterns for example

Also, since `SCA_IGNORE_LIST` is not used anymore, you could just include the values in the comprehension, and get rid of the variable altogether.